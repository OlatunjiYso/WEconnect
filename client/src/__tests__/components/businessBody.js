import React from "react";

import BusinessBody from '../../components/businessBody';

describe('All tests for business body component', () => {
 
  const props = {
    business: {
        address: "",
        about: "",
        city: "",
        state: "",
        phone: "",
        email: ""
    }
  };
  
    it('should be defined', () => {
      expect(BusinessBody).toBeDefined();
    });
    it('should render the body', () => {
      const { wrapper } = shallow(<BusinessBody {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
});
