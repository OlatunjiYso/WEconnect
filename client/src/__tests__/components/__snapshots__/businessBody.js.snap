// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for business body component should render the body 1`] = `undefined`;
